Cargo.lock

```diff
@@ -1053,13 +1030,11 @@ dependencies = [
 
 [[package]]
 name = "mime_guess"
-version = "2.0.0-alpha.6"
+version = "2.0.1"
```

Comment:
Well look at that, `multipart` was already using `mime_guess` so we don't even have any more dependencies :)